pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -55,6 +55,25 @@ func checkDeploymentHasEnvVar(t *testing.T, deployment *appsv1.Deployment, name
 	}
 }
 
+func checkDeploymentDoesNotHaveEnvVar(t *testing.T, deployment *appsv1.Deployment, name string) {
+	t.Helper()
+
+	var (
+		actualValue string
+		foundValue  bool
+	)
+	for _, envVar := range deployment.Spec.Template.Spec.Containers[0].Env {
+		if envVar.Name == name {
+			foundValue = true
+			actualValue = envVar.Value
+			break
+		}
+	}
+	if foundValue {
+		t.Errorf("router Deployment has unexpected %s setting: %q", name, actualValue)
+	}
```

Comment:
This function is equivalent to `checkDeploymentHasEnvVar(t, deployment, name, false, "")`.